File: pgm/ABC_Visual_Basic/ABC_393_A.vb
' Option Strict On
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S() As String = Console.ReadLine().Split()
        Const sick As String = "sick"
        Const fine As String = "fine"
        Const one As Integer = 1
        Const two As Integer = 2
        Const three As Integer = 3
        Const four As Integer = 4
        Dim S1 As String = S(0)
        Dim S2 As String = S(1)
        If S1 = sick AndAlso S2 = sick Then
            Console.WriteLine(one)
        ElseIf S1 = sick AndAlso S2 = fine Then
            Console.WriteLine(two)
        ElseIf S1 = fine AndAlso S2 = sick Then
            Console.WriteLine(three)
        Else
            Console.WriteLine(four)
        End If
        Return
    End Sub
End Class

File: pgm/ABC_Kotlin/ABC_393_A.kt
// import java.time.*

// import kotlin.math.*

const val one: Int = 1
const val two: Int = 2
const val three: Int = 3
const val four: Int = 4
const val sick: String = "sick"
const val fine: String = "fine"
const val space: String = " "

fun main() {
    val (S1: String, S2: String) = readln().split(space)
    if (S1 == sick && S2 == sick) {
        println(one)
    } else if (S1 == sick && S2 == fine) {
        println(two)
    } else if (S1 == fine && S2 == sick) {
        println(three)
    } else {
        println(four)
    }
    kotlin.system.exitProcess(0)
}